packages/dx-react-scheduler/src/plugins/day-view.jsx

```diff
@@ -0,0 +1,274 @@
+
+
+        <Template name="main">
+          <TemplateConnector>
+            {({
+              timeScale, appointments, startViewDate,
+              endViewDate, currentView, currentDate, dayScale,
+            }) => {
+              if (currentView !== viewName) return <TemplatePlaceholder />;
+              const intervals = calculateWeekDateIntervals(
+                appointments, startViewDate, endViewDate, [],
+              );
+              const rects = stateDateTableRef ? calculateRectByDateIntervals(
+                {
+                  growDirection: VERTICAL_APPOINTMENT_TYPE,
+                  multiline: false,
+                },
+                intervals,
+                getRectByDates,
+                {
+                  startViewDate,
+                  endViewDate,
+                  dayScale,
+                  timeScale,
+                  cellDuration,
+                  currentDate,
+                  cellElements: stateDateTableRef.querySelectorAll('td'),
+                },
+              ) : [];
+
+              const { appointmentPlaceholder: AppointmentPlaceholder } = this;
+              return (
+                <React.Fragment>
+                  <DateTable
+                    rowComponent={DateTableRow}
+                    cellComponent={DateTableCell}
+                    timeScale={timeScale}
+                    dayScale={dayScale}
+                    dateTableRef={this.dateTableRef}
+                  />
+                  <Container>
+                    {rects.map(({
+                      dataItem, type, ...geometry
+                    }, index) => (
+                      <AppointmentPlaceholder
+                        key={index.toString()}
+                        type={type}
+                        appointment={dataItem}
+                        style={getAppointmentStyle(geometry)}
+                      />
+                    ))}
+                  </Container>
+                </React.Fragment>
+              );
+            }}
+          </TemplateConnector>
+        </Template>
+      </Plugin>
+    );
+  }
+}
+
+DayView.propTypes = {
+  layoutComponent: PropTypes.func.isRequired,
+  navbarEmptyComponent: PropTypes.func.isRequired,
+  timePanelLayoutComponent: PropTypes.func.isRequired,
+  timePanelRowComponent: PropTypes.func.isRequired,
+  timePanelCellComponent: PropTypes.func.isRequired,
+  dayPanelLayoutComponent: PropTypes.func.isRequired,
+  dayPanelCellComponent: PropTypes.func.isRequired,
+  dayPanelRowComponent: PropTypes.func.isRequired,
+  dateTableLayoutComponent: PropTypes.func.isRequired,
+  dateTableRowComponent: PropTypes.func.isRequired,
+  dateTableCellComponent: PropTypes.func.isRequired,
+  containerComponent: PropTypes.func.isRequired,
+  startDayHour: PropTypes.number,
+  endDayHour: PropTypes.number,
+  cellDuration: PropTypes.number,
+  intervalCount: PropTypes.number,
+  viewName: PropTypes.string,
+};
+
+DayView.defaultProps = {
+  viewName: 'Day',
```

Comment:
Should we use `Symbol` here(and for other views)? It allows us to avoid creating views with the same name.

Reply:
No, because `viewName` is used in ViewSwitcher in the dropdown menu